package.json

```diff
@@ -7,7 +7,7 @@
   "version": "1.35.0",
   "description": "A framework to support application development with Terra components",
   "engines": {
-    "node": ">=8.9.2 <11"
+    "node": ">=10"
```

Comment:
Do we want this to be ">=10 <13"? Some of our projects restrict to just 10 while others don't. How do we determine whether or not to include `<13`?

Reply:
Here's my thinking:

A packages 'engines' node version should reflect the version of node that the packaged module can run on. 

We've been dropping support for the old versions of node which is fine, but i don't know of anything that would prevent the @cerner/terra-application package from running on node 14. The git repo itself won't run on node 14 because of the fibers transitive dependency but the released package doesn't have that issue.